core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -198,6 +198,47 @@ public static long parseSpaceSize(String spaceSize) {
   }
 
   /**
+   * Parses a String size to Milliseconds.
+   *
+   * @param timeSize the size of a time, e.g. 1M, 5H, 10D
+   * @return the time size in milliseconds
+   */
+  public static long parseTimeSize(String timeSize) {
+    double alpha = 0.0001;
+    String ori = timeSize;
+    String end = "";
+    int index = timeSize.length() - 1;
+    while (index >= 0) {
+      if (timeSize.charAt(index) > '9' || timeSize.charAt(index) < '0') {
+        end = timeSize.charAt(index) + end;
+      } else {
+        break;
+      }
+      index--;
+    }
+    timeSize = timeSize.substring(0, index + 1);
+    double ret = Double.parseDouble(timeSize);
+    end = end.toLowerCase();
+    if (end.isEmpty() || end.equalsIgnoreCase("ms")
+            || end.equalsIgnoreCase("millisecond")) {
```

Comment:
indent is off here. We use google style and line-break indents 4 spaces.
See https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent